vars/Debian.yml

```diff
@@ -1,10 +1,5 @@
 # SPDX-License-Identifier: MIT
 ---
-# Put internal variables here with Debian 10 specific values.
-
-__certificate_packages:
-  - python-pyasn1
-  - python-cryptography
-  - python-dbus
```

Comment:
FTR: The python3 packages have existed for ages, at least since stretch. [cryptography](https://packages.debian.org/search?keywords=python3-cryptography), [pyasn1](https://packages.debian.org/search?keywords=python3-pyasn1), [dbus](https://packages.debian.org/search?keywords=python3-dbus)